website/src/word-panel.tsx

```diff
@@ -75,19 +77,77 @@ export const WordPanel = (p: {
         </Button>
         <h1>Selected word:</h1>
         <h2 className={css.cherHeader}>{p.segment.source}</h2>
-        {phonetics}
-        <MorphemicSegmentation
-          segments={p.segment.segments}
-          onOpenDetails={p.onOpenDetails}
-          level={p.viewMode}
-          tagSet={p.tagSet}
+        <CollapsiblePanel
+          title={"Phonetics"}
+          content={<>{phonetics}</>}
+          icon={
+            <MdRecordVoiceOver
+              size={24}
+              className={css.wordPanelButton.colpleft}
+            />
+          }
         />
-        {translation.length ? <div>&lsquo;{translation}&rsquo;</div> : <br />}
-        <br />
-        <p>{p.segment.commentary}</p>
+
+        <CollapsiblePanel
+          title={"Word Parts"}
+          content={
+            <>
+              <MorphemicSegmentation
+                segments={p.segment.segments}
+                onOpenDetails={p.onOpenDetails}
+                level={p.viewMode}
+                tagSet={p.tagSet}
+              />
+              {translation.length ? (
+                <div>&lsquo;{translation}&rsquo;</div>
+              ) : (
+                <br />
+              )}
+            </>
+          }
+          icon={
+            <IoEllipsisHorizontalCircle
+              size={24}
+              className={css.wordPanelButton.colpleft}
+            />
+          }
+        />
+        {p.segment.commentary ? (
+          <CollapsiblePanel
+            title={"Commentary"}
+            content={<>{p.segment.commentary}</>}
+            icon={
+              <MdNotes size={24} className={css.wordPanelButton.colpleft} />
+            }
+          />
+        ) : null}
       </div>
     )
   } else {
     return <p>No word has been selected</p>
   }
 }
+
+const CollapsiblePanel = (p: {
+  title: String
+  content: JSX.Element
+  icon: JSX.Element // Note : this is supposed to be an IconType
+}) => {
+  const disclosure = useDisclosureState({ visible: true })
+  return (
+    <div className={css.collPanel}>
+      <Disclosure {...disclosure} className={css.collPanelButton}>
```

Comment:
I didn't think of that but yeah. The header for the panel and the aria label would be the same, but I don't think that will be an issue.

Reply:
Yeah so `aria-label` is only for things like screen readers and it's the same as the section name but without the two icons potentially adding noise, or making the browser recompute the label because the container contents change (when the arrow icon switches).